src/ast/passes/printer.cpp

```diff
@@ -321,12 +321,12 @@ void Printer::visit(Unroll &unroll)
   out_ << indent << "unroll" << std::endl;
 
   ++depth_;
-  unroll.expr->accept(*this);
+  unroll.expr().accept(*this);
```

Comment:
the reason we need `operator()` here and not for `unroll.stmts` is the binding in the latter implicitly converts the `ref`?

Reply:
yeah, exactly